app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -62,6 +83,11 @@ def create
 
     private
 
+    def get_buffer_range(current_buffer_index:, buffer_size:, file_size:)
+      buffer_end = [current_buffer_index + buffer_size, file_size].min
+      return "bytes=#{current_buffer_index}-#{buffer_end}"
```

Comment:
Style suggestions, I'd remove `get_` from the method name, also remove the `return`

```suggestion
    def buffer_range(current_buffer_index:, buffer_size:, file_size:)
      buffer_end = [current_buffer_index + buffer_size, file_size].min
      "bytes=#{current_buffer_index}-#{buffer_end}"
```

Reply:
Done